### DISCUSSION TITLE
First LTS release

### DISCUSSION THREAD

**Discussion by imdhemy** [imdhemy/laravel-in-app-purchases#122] [General]
This version is a long-term support release and a complete rewrite of the package. There are many breaking
changes that are not backward compatible. You can still use the pre-release version `0.x` for a while until an
upgrade guide to `1.x` is available. It will be available on the [documentation site](https://imdhemy.com/laravel-iap-docs/).

This version supports receipt validation, server notifications, and mocking receipts 🎉. The next versions will
support App Store notifications v.2.

**Sponsors** 🙌  ❤️ 

This release is sponsored by
- @abdelrahman-waziry
- @iabdullahbeker

**Become a sponsor** 🥇 

https://github.com/sponsors/imdhemy

<hr /><em>This discussion was created from the release <a href='https://github.com/imdhemy/laravel-in-app-purchases/releases/tag/1.0.0'>First LTS release</a>.</em>

**Comment by vesper8** [comment]
@imdhemy Any ETA on when an upgrade guide will be provided?

**Reply by imdhemy** [reply]
@vesper8 
Now, I'm focusing on supporting App Store notifications v2, after that I may work on the upgrade guide. 

If you are in a hurry, the breaking changes include:

- updating the webhook URL.
- Changes in the value objects, some of them are removed.
- Access to the original raw data as an array.

I will publish a complete guide ASAP, but I'm still not sure when. I will keep you posted.

**Reply by vesper8** [reply]
@imdhemy Thank you. Not in a hurry and I will wait until you post a guide before attempting the migration. Thanks for your continued hard work and support!

**Reply by imdhemy** [reply]
@vesper8 My pleasure! 🙌

**Reply by imdhemy** [reply]
@vesper8 

I summarized the changes [here](https://imdhemy.com/laravel-iap-docs/docs/upgrade). Feel free to ask any questions during your upgrade process. In case you faced any problems I will be here for support.
